content/news/2021-XX-XX-keyfinder-gplv3.md

```diff
@@ -0,0 +1,30 @@
+title: "New in 2.3: KeyFinder support and license change to GPL v3"
+authors: Jan Holthuis
+tags: license, key detection, 2.3, beta
+status: draft
+
+Starting with version 2.3, Mixxx will be licensed under the terms of the [GNU General Public License (GNU GPL) version 3](https://www.gnu.org/licenses/rms-why-gplv3.html).
+This step is necessary because it allows us to integrate support for the excellent [KeyFinder library](https://github.com/mixxxdj/libKeyFinder) by Ibrahim Sha'ath.
```

Comment:
```suggestion
This step is necessary because it allows us to integrate support for the excellent [KeyFinder library](https://github.com/mixxxdj/libkeyfinder) by Ibrahim Sha'ath.
```